app/assets/stylesheets/application.css.scss

```diff
@@ -188,18 +189,63 @@ td {
 	vertical-align: top;
 }
 
+
+.header {
+  background-color: black;
+  color: white;
+  padding: 10px;
+  margin-left:10px;
+  margin-top:20px;
+  margin-bottom:20px;
```

Comment:
Generally if you are specifying > 2 margin properties you should use the shorthand:
`margin: top right bottom left;`
So this would be `margin: 20px 0px 20px 10px;`
Less code